modules/server/include/connection.h

```diff
@@ -71,11 +72,10 @@ class Connection : public std::enable_shared_from_this<Connection> {
     std::map<TopicId, std::unique_ptr<Topic>> _topics;
     std::unique_ptr<ghoul::io::Socket> _socket;
     std::thread _thread;
+    std::mutex _mutex;
```

Comment:
Somehow, I feel like this mutex lock doesn't belong in this file... It feels like the connection class gets polluted from the multithreading in a way it doesn't have to be 🤔